### ISSUE TITLE
How to implement https instead of http ? Cannot find any example on this.

### ISSUE THREAD

**Issue by adibro500** [matt-42/lithium#9]
Please provide an example to implement https. 
Thanks,
adibro

**Comment by matt-42** [comment]
It's not yet implemented. I'll add this feature to my todolist. In the meantime you can use nginx as a https gateway that relay https requests to Http requests

**Comment by matt-42** [comment]
I implemented HTTPS in 540e2ee1267eeb5d68845d96c61fff533b3377dd. 
To activate it, you have to pass the `s::ssl_key` and `s::ssl_certificate` options to `http_serve`:

```c++
http_serve(my_api, 443, s::ssl_key = "./server.key", s::ssl_certificate = "./server.crt");
```
You'll also have to link you program with openssl.
